src/components/buttonnavigation/ButtonNavigationCSS.js

```diff
@@ -0,0 +1,31 @@
+import { colors } from "../colors/index";
+
+export const ButtonNavigationCSS = `
+    position: absolute;
+    padding: 0;
+    background: none;
+    border: none;
+
+    font-family: 'HK Grotesk', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 1.5rem;
+    line-height: 48px;
+    /* identical to box height, or 200% */
+
+    display: flex;
+    align-items: center;
+    text-transform: capitalize;
+
+    /* white */
```

Comment:
nit: what is this comment for?

Reply:
not sure i think it was just for me to know when I was working on it but ill remove it